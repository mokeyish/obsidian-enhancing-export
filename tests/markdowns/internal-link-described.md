[[#Some complex section|with custom description]]
