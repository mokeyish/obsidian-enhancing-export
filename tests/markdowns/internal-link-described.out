Pandoc
  Meta { unMeta = fromList [] }
  [ Para
      [ Link
          ( "" , [] , [] )
          [ Str "with"
          , Space
          , Str "custom"
          , Space
          , Str "description"
          ]
          ( "#Some-complex-section" , "" )
      ]
  ]
